startGame.js

```diff
@@ -0,0 +1,29 @@
+document.getElementById("button").addEventListener('click', function(){
+    setTimeout(startGame, 3000)
+    Timer()
+})
+
+ function startGame() {
+ }
+
+function countdown(number) {
+    return number - 1
+}
+
+function Timer() {
+    time = 4
+    var timer = document.getElementById("timer")
+    createTimer()
+    function createTimer() {
```

Comment:
you have pt a function inside a function. Bring outside.

Reply:
done